actionpack/lib/action_dispatch/http/parameters.rb

```diff
@@ -51,12 +51,12 @@ def path_parameters
       # Invoca Patch
       def strip_string_params!(value_to_strip)
         case value_to_strip
-          when String
-            value_to_strip.strip!
-          when Hash
-            value_to_strip.each{ |key, value| strip_string_params!(value) if !key.respond_to?(:to_sym) || key.to_sym.not_in?(do_not_strip_string_parameters) }
-          when Array
-            value_to_strip.each{ |value| strip_string_params!(value) }
+        when String
+          value_to_strip.strip!
+        when Hash
+          value_to_strip.each { |key, value| strip_string_params!(value) if !key.respond_to?(:to_sym) || !do_not_strip_string_parameters.include?(key.to_sym) }
```

Comment:
Good catch. I was thinking that was a boolean when actually it's an array, or nil. I backed out that change and added the spaces: https://github.com/Invoca/rails/commit/a96974e48dd354e7ca3bc4a98383ac6f76eebf90